client/src/main/scala/io/delta/sharing/client/util/ConfUtils.scala

```diff
@@ -111,6 +123,44 @@ object ConfUtils {
     maxDur
   }
 
+  def asyncQueryPollIntervalMillis(conf: Configuration): Long = {
```

Comment:
Why do we implement each twice? Are they needed?

Reply:
I am following the existing pattern, looks like we want to support 2 different type of config system.